doc/developer.texinfo

```diff
@@ -2446,34 +2436,99 @@ GNU Texinfo Manual (@uref{http://www.gnu.org/software/texinfo/manual/texinfo/tex
 
 @need 1000
 @item
-Build everything to be distributed and copy it to the temporary staging area
-@c not @uref{...} because we don't want robots to notice and crawl the link
-(http://plse.cs.washington.edu/daikon/staging-daikon/).
-Part of this step is to build a set of publication abstracts.  If there are
-any failures, then investigate and correct as necessary.
-The final output of this step
-will be a list of files that were added/removed since the
-last release.
-If any of these differences is unexpected, then investigate and correct the
-difference, and redo the step.
+Once you have successfully reached this point, you are ready to make a release
+candidate and begin testing it. Any source changes required to fix a problem
+encountered above must be committed to the repository now.
+Verify doc/CHANGES is up-to-date and there are no uncommitted files.
```

Comment:
It's not clear that this is a description of the steps that follow.  I thought it was something that I had to do now, and I wondered how to do it.  This may just be an issue with the way the instructions are written; I was expecting a list of steps to perform rather than a description of what will come, followed by a list of steps to perform.  maybe it would be clearer to say (rather than "Verify ..."), "The following steps verify ...".